multiversx_sdk_cli/cli_wallet.py

```diff
@@ -119,14 +119,14 @@ def generate_pem(args: Any):
 
     if ask_mnemonic:
         mnemonic = input("Enter mnemonic:\n").rstrip().replace("\n", "")
-        print(mnemonic, 'asad')
         mnemonic = Mnemonic(mnemonic)
         secret_key = mnemonic.derive_key(index)
         pubkey = secret_key.generate_public_key()
     else:
         mnemonic = Mnemonic.generate()
-        secret_key = mnemonic.derive_key(index)
-        pubkey = secret_key.generate_public_key()
+    
+    secret_key = mnemonic.derive_key(index)
```

Comment:
Now I see that, above, `mnemonic = input(...)` can be `mnemonic_str = input(...)`, since below it, the same variable is used, but it has a different type - works, of course, but can be adjusted.

Reply:
Will do it in the next pr.